### DISCUSSION TITLE
Impressions for channels with no impressions

### DISCUSSION THREAD

**Discussion by KristiGourlay** [google/meridian#296] [Q&A]
How do you handle media channels with no impressions? With other MMMs, you're supposed to input the spend. Is this the same with Meridian? Or should I just not input impressions for that channel? 

For reference, my case is referring to Rakuten.

**Comment by brendalewisprice** [comment]
Yes, you can input channels that have spend but no impressions into your model. 

You do that by specifying the type of channel in the `CoordToColumns` mapping. Just set that channel to "spend" instead of "impressions". 

For example, say you have impressions and spend for Channel0 but only spend for Channel1, then the syntax would be
 ```
coord_to_columns = load.CoordToColumns(
     ...
     media=[
         'Channel0_impression',
         'Channel1_spend',

     ],
     media_spend=[
         'Channel0_spend',
         'Channel1_spend',
     ],
)
```
